crates/bevy_pbr/src/fullscreen_material.rs

```diff
@@ -0,0 +1,233 @@
+//! This is mostly a pluginify version of the custom_post_processing example
```

Comment:
Move to bevy_core_pipeline?

Reply:
Yep, that's one of the things I'm planning on doing before moving it out of draft. It feels weird to put it there, but I don't see any other crate that would make sense for something dimension agnostic